salt/elife-bot/config/opt-elife-bot-digest.cfg

```diff
@@ -10,10 +10,10 @@ medium_content_pattern: {figure}{title}{summary}<hr/>{body}{footer}
 medium_content_format: html
 medium_license: 
 # Medium application settings
-medium_application_client_id:
-medium_application_client_secret:
+medium_application_client_id: {{ pillar.elife_bot.medium.application_client_id }}
```

Comment:
use the `yaml` filter like in here:
https://github.com/elifesciences/journal-formula/blob/master/salt/journal/config/srv-journal-app-config-parameters.yml#L22
so that the `None` value gets rendered to a proper null rather than the string `None`, which I think would be parsed as a string by the application

Reply:
I'm ok with a blank string, and an explicit `null` is not a requirement. Cool example though!